syntax.php

```diff
@@ -201,13 +201,20 @@ function render($mode, &$renderer, $data) {
       }
       return true;
     }
+    
+    if ($mode == 'odt') {
```

Comment:
What about turning this "if" into an "else"? It seems to me it would be even more generic, because it would provide a relevant downgraded rendering for each none html mode.

What do you think? Do you have in mind drawbacks I could be missing?